src/sarif_converter.ts

```diff
@@ -127,9 +134,15 @@ const getResults = (suggestions: ISarifSuggestions) => {
             },
           },
         },
-      ],
+      ]
     };
 
+    if (suggestion.fingerprints) {
+      result.fingerprints = {};
+      suggestion.fingerprints.forEach(fingerprinting => {
+        (result.fingerprints as any)[fingerprinting.version] = fingerprinting.fingerprint;
```

Comment:
The Sarif format wants fingerprints of type `Record<string, string>`: https://docs.oasis-open.org/sarif/sarif/v2.0/csprd02/sarif-v2.0-csprd02.html#_Toc10127845
Since the version returned by the API Server is a number at the moment, be sure to wrap it in a string (I know that all the js arrays are basically objects using numbers as keys, but let's be explicit about it and use something like  ``[`${fingerprinting.version}`]``).

Reply:
valid point :) changed...